x-pack/docs/en/rest-api/security/bulk-update-api-keys.asciidoc

```diff
@@ -2,4 +2,367 @@
+
+Which returns the response:
+
+[source,console-result]
+----
+{
+  "updated": [
+    "VuaCfGcBCdbkQm-e5aOx",
+    "H3_AhoIBA9hmeQJdg7ij"
+  ],
+  "noops": []
+}
+----
+
+The API keys' effective permissions after the update will the same as the <<security-api-bulk-update-api-keys-examples-user-permissions, owner user's>>:
+
+[source,js]
+--------------------------------------------------
+{
+  "cluster": ["all"],
+  "index": [
+    {
+      "names": ["*"],
+      "privileges": ["all"]
+    }
+  ]
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+For the next example, assume that the owner user's permissions have changed from <<security-api-bulk-update-api-keys-examples-user-permissions, the original permissions>> to:
+
+[source,js]
+--------------------------------------------------
+{
+  "cluster": ["manage_security"],
+  "index": [
+    {
+      "names": ["*"],
+      "privileges": ["read"]
+    }
+  ]
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+The following request auto-updates the snapshot of the user's permissions associated with the two API keys.
+
+[source,console]
+----
+POST /_security/api_key/_bulk_update
+{
+  "ids": [
+    "VuaCfGcBCdbkQm-e5aOx",
+    "H3_AhoIBA9hmeQJdg7ij"
+  ]
+}
+----
+// TEST[skip:api key ids not available]
+
+Which returns the response:
+
+[source,console-result]
+----
+{
+  "updated": [
+    "VuaCfGcBCdbkQm-e5aOx",
+    "H3_AhoIBA9hmeQJdg7ij"
+  ],
+  "noops": []
+}
+----
+
+Resulting in the following effective permissions for both API keys:
+
+[source,js]
+--------------------------------------------------
+{
+  "cluster": ["manage_security"],
+  "index": [
+    {
+      "names": ["*"],
+      "privileges": ["read"]
+    }
+  ]
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+If any API keys fail to update, error details are included in the `errors` field.
+For example:
+
+[[security-api-bulk-update-api-keys-examples-errors]]
+[source,js]
+--------------------------------------------------
+{
+  "updated": ["VuaCfGcBCdbkQm-e5aOx"],
+  "noops": [],
+  "errors": { <1>
+    "count": 3,
+    "details": {
+       "g_PqP4IBcBaEQdwM5-WI": { <2>
+         "type": "resource_not_found_exception",
+         "reason": "no API key owned by requesting user found for ID [g_PqP4IBcBaEQdwM5-WI]"
+       },
+       "OM4cg4IBGgpHBfLerY4B": {
+         "type": "illegal_argument_exception",
+         "reason": "cannot update invalidated API key [OM4cg4IBGgpHBfLerY4B]"
+       },
+       "Os4gg4IBGgpHBfLe2I7j": {
+         "type": "exception",
+         "reason": "error occurred while updating API key",
+         "caused_by": { <3>
+           "type": "illegal_argument_exception",
+           "reason": "security index failure"
+         }
+       }
+    }
+  }
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+<1> Details about any errors encountered.
+This field is not present in the response when `count` is 0.
```

Comment:
```suggestion
<1> This field is not present in the response when the error `count` is 0.
```
Since we have an actual field named `details`, saying this field is "Details about ..." is a bit awkward and redundant.